qutebrowser/browser/hints.py

```diff
@@ -240,7 +243,18 @@ def yank(self, url, context):
         if url.scheme() == 'mailto':
             flags |= QUrl.RemoveScheme
         urlstr = url.toString(flags)
-        utils.set_clipboard(urlstr, selection=sel)
+
+        new_content = urlstr
+
+        # only second and consecutive yanks are to append to the clipboard
+        if context.rapid and context.num > 1:
+            try:
+                old_content = utils.get_clipboard(selection=sel)
+            except utils.ClipboardEmptyError:
+                pass
+            else:
+                new_content = old_content + '\n' + new_content
```

Comment:
I think this should be `os.linesep.join(old_content, new_content)` for our windows and mac using friends.